src/containers/stage-selector.jsx

```diff
@@ -51,6 +55,76 @@ class StageSelector extends React.Component {
             });
         }
     }
+    handleBackdropUpload (e) {
+        const thisFileInput = e.target;
+        let thisFile = null;
+        const reader = new FileReader();
+        reader.onload = () => {
```

Comment:
Can we move some of this to a lib or something so it isn't just duplicated from the other place?